packages/cli/src/commands/cluster/install/setupInboundNetworking.ts

```diff
@@ -136,38 +137,111 @@ export async function setupInboundNetworking(
+                    context,
+                    env: {
+                      ...process.env,
+                      VAULT_ADDR: `http://127.0.0.1:${ctx.vaultProxyPort}`,
+                      VAULT_TOKEN: vaultRootToken,
+                    },
+                    environment,
+                    region,
+                    module: MODULES.KUBE_INGRESS_NGINX,
+                    initModule: true,
+                    hclIfMissing: await Bun.file(
+                      kubeNginxIngressTerragruntHcl
+                    ).text(),
+                    inputUpdates: {
+                      ingress_domains: defineInputUpdate({
+                        schema: z.array(z.string()),
+                        update: () => [kubeDomain],
+                      }),
+                      sla_level: defineInputUpdate({
+                        schema: z.number(),
+                        update: () => 1,
+                      }),
+                    },
+                    postDeployInputUpdates: {
+                      sla_level: defineInputUpdate({
+                        schema: z.number().optional(),
+                        update: () => undefined,
+                      })
+                    }
+                  }),
+                  {
+                    title: "Resetting Cert Manager",
+                    task: async () => {
+                      let attempts = 0;
+                      const maxAttempts = 10;
+                      const retryDelay = 90000;
+
+                      const kc = new k8s.KubeConfig();
+                      kc.loadFromDefault();
+
+                      const customApi = kc.makeApiClient(k8s.CustomObjectsApi);
+                      const appsApi = kc.makeApiClient(k8s.AppsV1Api);
+
+                      while (attempts < maxAttempts) {
+                        try {
+                          // Get Certificate resources
+                          const result = await customApi.listClusterCustomObject({
+                            group: 'cert-manager.io',
+                            version: 'v1',
+                            plural: 'certificates'
+                          });
+
+                          const certificates = (result?.body as unknown as { items?: unknown[] })?.items || [];
+                          // Check for non-Ready certificates
+                          const hasNonReadyCerts = certificates.some((cert: unknown) => {
+                            const typedCert = cert as { status?: { conditions?: unknown[] } };
+                            const conditions = typedCert?.status?.conditions || [];
+                            return !conditions.some((c: unknown) => {
+                              const condition = c as { type: string; status: string };
+                              return condition.type === 'Ready' && condition.status === 'True';
+                            });
+                          });
+
+                          if (!hasNonReadyCerts) {
```

Comment:
I believe there is a logic bug. What if this task runs before the cert has been created by the other task?